infrastructure/test-restore-backup.sh

```diff
@@ -0,0 +1,115 @@
+ssh "$SSH_USER@$SSH_HOST" "rm -rf $RESTORE_DIRECTORY/mongo"
+ssh "$SSH_USER@$SSH_HOST" "rm -rf $RESTORE_DIRECTORY/minio"
+ssh "$SSH_USER@$SSH_HOST" "rm -rf $RESTORE_DIRECTORY/metabase"
+ssh "$SSH_USER@$SSH_HOST" "rm -rf $RESTORE_DIRECTORY/vsexport"
+ssh "$SSH_USER@$SSH_HOST" "rm -rf $RESTORE_DIRECTORY/metabase"
+
+ssh "$SSH_USER@$SSH_HOST" "rsync -av --delete --progress $SSH_USER@$BACKUP_HOST:$REMOTE_DIR/elasticsearch $RESTORE_DIRECTORY"
+ssh "$SSH_USER@$SSH_HOST" "rsync -av --delete --progress $SSH_USER@$BACKUP_HOST:$REMOTE_DIR/mongo/hearth-dev-$LABEL.gz $RESTORE_DIRECTORY/mongo/"
+ssh "$SSH_USER@$SSH_HOST" "rsync -av --delete --progress $SSH_USER@$BACKUP_HOST:$REMOTE_DIR/mongo/openhim-dev-$LABEL.gz $RESTORE_DIRECTORY/mongo/"
+ssh "$SSH_USER@$SSH_HOST" "rsync -av --delete --progress $SSH_USER@$BACKUP_HOST:$REMOTE_DIR/mongo/user-mgnt-$LABEL.gz $RESTORE_DIRECTORY/mongo/"
+ssh "$SSH_USER@$SSH_HOST" "rsync -av --delete --progress $SSH_USER@$BACKUP_HOST:$REMOTE_DIR/mongo/application-config-$LABEL.gz $RESTORE_DIRECTORY/mongo/"
+ssh "$SSH_USER@$SSH_HOST" "rsync -av --delete --progress $SSH_USER@$BACKUP_HOST:$REMOTE_DIR/mongo/metrics-$LABEL.gz $RESTORE_DIRECTORY/mongo/"
+ssh "$SSH_USER@$SSH_HOST" "rsync -av --delete --progress $SSH_USER@$BACKUP_HOST:$REMOTE_DIR/mongo/webhooks-$LABEL.gz $RESTORE_DIRECTORY/mongo/"
+ssh "$SSH_USER@$SSH_HOST" "rsync -av --delete --progress $SSH_USER@$BACKUP_HOST:$REMOTE_DIR/mongo/performance-$LABEL.gz $RESTORE_DIRECTORY/mongo/"
+ssh "$SSH_USER@$SSH_HOST" "rsync -av --delete --progress $SSH_USER@$BACKUP_HOST:$REMOTE_DIR/influxdb/$LABEL $RESTORE_DIRECTORY/influxdb"
+ssh "$SSH_USER@$SSH_HOST" "rsync -av --delete --progress $SSH_USER@$BACKUP_HOST:$REMOTE_DIR/minio/ocrvs-$LABEL.tar.gz $RESTORE_DIRECTORY/minio/"
+ssh "$SSH_USER@$SSH_HOST" "rsync -av --delete --progress $SSH_USER@$BACKUP_HOST:$REMOTE_DIR/metabase/ocrvs-$LABEL.tar.gz $RESTORE_DIRECTORY/metabase/"
+ssh "$SSH_USER@$SSH_HOST" "rsync -av --delete --progress $SSH_USER@$BACKUP_HOST:$REMOTE_DIR/vsexport/ocrvs-$LABEL.tar.gz $RESTORE_DIRECTORY/vsexport/"
+
+# Restore
+echo "Restoring"
+# Uncomment if your SSH user is not root
+#
+# ssh "$SSH_USER@$SSH_HOST" "echo $SUDO_PASSWORD | sudo -S rm -rf /data/backups/elasticsearch && mv $RESTORE_DIRECTORY/elasticsearch /data/backups/"
+# else 
+ssh "$SSH_USER@$SSH_HOST" "rm -rf /data/backups/elasticsearch && mv $RESTORE_DIRECTORY/elasticsearch /data/backups/"
+
+ssh "$SSH_USER@$SSH_HOST" "docker service update --force --update-parallelism 1 --update-delay 30s opencrvs_elasticsearch"
+echo "Waiting 2 mins for elasticsearch to restart."
+echo
+sleep 120
+# Uncomment if your SSH user is not root
+#
+# ssh "$SSH_USER@$SSH_HOST" "echo $SUDO_PASSWORD | sudo -S bash /opt/opencrvs/infrastructure/emergency-restore-metadata.sh --label=$LABEL --replicas=$REPLICAS --backup-dir=$RESTORE_DIRECTORY"
+# else 
+ssh "/opt/opencrvs/infrastructure/emergency-restore-metadata.sh --label=$LABEL --replicas=$REPLICAS --backup-dir=$RESTORE_DIRECTORY"
```

Comment:
@rikukissa after this has finished, the only way to make the search service able to access the new Elastic is to re-deploy. I get connection refused.  I think it is because Elastic is configured with a temporary password used in production and we need to run the setup-elasticsearch-users again maybe ? It would be nice if it would just work in this script without a redeploy